agent/fw_lumen.c

```diff
@@ -139,6 +147,12 @@ NR_PHP_WRAPPER_END
 /*
  * Exception handling logic. Wraps the function that routes
  * exceptions to their respective handlers
+ *
+ * In this case, `nr_txn_set_path` is called after `NR_PHP_WRAPPER_CALL` with
```

Comment:
I'm not seeing how `nr_txn_set_path` is called in this function after `NR_PHP_WRAPPER_CALL`.  Should it instead say the exception is named?

Reply:
Thank you, good catch.
Yes, nr_txn_set_path is called before NR_PHP_WRAPPER_CALL [here](https://github.com/newrelic/newrelic-php-agent/blob/8a3028d7517fafdf1ac9b9c639827aac49302367/agent/fw_lumen.c#L183).
So it also needs to be called before for oapi.
Please see: [63e8899](https://github.com/newrelic/newrelic-php-agent/pull/615/commits/63e889962206aea00c268c65faacf511a301eddd)